### PR TITLE
add io.github.mideb.digitales_register

### PR DESCRIPTION
**Pull request by miDeb**
digitales_register is a client for https://digitalesregister.it where students and parents can seen their homework, grades, schedule, and other information. To my knowledge this service is only used in the province of South Tyrol (Italy) by German speaking schools.

I wrote all user-facing descriptions in German because the app doesn't support other languages at all.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by miDeb** [comment]
bot, build io.github.mideb.digitales_register

**Comment by miDeb** [comment]
I guess I should add the .metainfo.xml, .desktop and the icon to the upstream project.

File: io.github.mideb.digitales_register.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - name: digitales_register
+    buildsystem: simple
+    build-commands:
+      - ls -R digitales_register
+      - cp -r digitales_register /app/digitales_register
+      - chmod +x /app/digitales_register/digitales_register
+      - ln -s /app/digitales_register/digitales_register /app/bin/digitales_register
+      - install -Dm644 io.github.mideb.digitales_register.desktop /app/share/applications/io.github.mideb.digitales_register.desktop
+      - install -Dm644 io.github.mideb.digitales_register.metainfo.xml /app/share/appdata/io.github.mideb.digitales_register.metainfo.xml
+      - install -Dm644 icon.png /app/share/icons/hicolor/scalable/apps/io.github.mideb.digitales_register.svg
```
Comment (hfiguiere) [inline io.github.mideb.digitales_register.yaml:46]: png isn't svg.

Comment (miDeb) [reply io.github.mideb.digitales_register.yaml]: Thanks, should be fixed now.

**Comment by miDeb** [comment]
bot, build io.github.mideb.digitales_register

**Comment by nedrichards** [comment]
to the extent that I can test this without being a student lgtm

**Comment by miDeb** [comment]
How long does it usually take for a new app to be published on flathub?

**Pull request closed without merging**